Tasks/Maven/maventask.ts

```diff
@@ -202,9 +202,6 @@ async function execBuild() {
             if (userRunFailed || codeAnalysisFailed) {
                 tl.setResult(tl.TaskResult.Failed, "Build failed."); // Set task failure
             }
-            else {
```

Comment:
The Gradle task seems to use slightly different logic. Did you check this does not happen on Gradle?